prep-javascript-variables/index.html

```diff
@@ -8,16 +8,16 @@
       const fullName = 'Andrew Owyang';
       console.log(fullName);
       console.log(typeof fullName);
-      var numberOfPets = 0;
+      const numberOfPets = 0;
       console.log(numberOfPets);
       console.log(typeof numberOfPets);
-      var likesVegetables = true;
+      const likesVegetables = true;
       console.log(likesVegetables);
       console.log(typeof likesVegetables);
-      var empty = null;
+      const empty = null;
       console.log(empty);
       console.log(typeof empty);
-      var unset;
+      const unset;
```

Comment:
Since this variable didn't get assigned a value, it should be initialized with `let`, since we would need to eventually assign it a value in the future. We will discuss more on the difference between `const` and `let` in our Q&A, but wanted to point it out now!